queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -88,6 +90,13 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
     if business.legal_type == Business.LegalTypes.COOP:
         _update_cooperative(dissolution_filing, business, filing_rec, dissolution_type)
 
+    # remove dissolution flag if business can be withdrawn
```

Comment:
Think this should be done in AR filer processor instead of dissolution processor...

Reply:
like Hongjing has mentioned, please move your code to AR processor.

Reply:
Updated